data/scopes/defaults.ini

```diff
@@ -36,3 +36,9 @@ ldap_number_filter = "(|(telephoneNumber=%)(mobile=%)(homePhone=%))"
 ldap_name_filter = "(|(cn=%)(o=%))"
 vlan_id_phone = ""
 vlan_id_pcport = ""
+cap_ringtone_count = ""
+cap_ringtone_blacklist = ""
+background_file = ""
+screensaver_file = ""
+cap_background_file = ""
+cap_screensaver_file = ""
```

Comment:
We have to set safe defaults for the use case "phone without the model" configuration